### DISCUSSION TITLE
Add a file when uploading a folder

### DISCUSSION THREAD

**Discussion by LexKul** [drakkan/sftpgo#1320] [General]
Is it possible using the actions to do: in the web client, when choosing a folder to upload, add a zip file and upload a zip folder with files and the added zip-file?
Or maybe there are some ideas how to do this?
Thanks!

**Comment by drakkan** [comment]
Hello, thanks for contacting us. Could you please clarify what you’d like to achieve? The more details you provide, the better. An example would be great. We are always interested in learning about new use-cases and features that our users could benefit from.
